sdk/src/camera_96tof1.cpp

```diff
@@ -15,7 +15,13 @@ static const std::string skCustomMode = "custom";
 Camera96Tof1::Camera96Tof1(std::unique_ptr<aditof::DeviceInterface> device)
     : m_specifics(std::make_shared<aditof::Camera96Tof1Specifics>(
           aditof::Camera96Tof1Specifics(this))),
-      m_device(std::move(device)), m_devStarted(false) {}
+      m_device(std::move(device)), m_devStarted(false) {
+
+    // initialize range values with the default data for revision C
```

Comment:
We assume the default is revision C in two places: here and in Camera96Tof1Specifics. Let's just do it once. Interrogate the revision in Camera96Tof1Specifics and use it here.
Also it would be nice to create a table with range values for each revision. Makes things clearer and you won't even need the setRangeValues() anymore.

Reply:
changed, thanks